packages/@coorpacademy-progression-engine/src/types.js

```diff
@@ -25,6 +25,7 @@ export type Answer = Array<string>;
 
 export type MICROLEARNING = 'microlearning';
 export type LEARNER = 'learner';
+export type EXTERNAL = 'external';
 
 export type Engines = MICROLEARNING | LEARNER;
```

Comment:
```suggestion
export type Engines = MICROLEARNING | LEARNER | EXTERNAL;
```

C'est étrange que la CI passe.

Reply:
Inquiétant même

Reply:
les tests et flow passaient en local

Reply:
ok, ce type ne sert à rien... tant pis.